pandas/plotting/_core.py

```diff
@@ -833,6 +833,23 @@ def _post_plot_logic(self, ax, data):
         ax.set_ylabel(pprint_thing(y))
         ax.set_xlabel(pprint_thing(x))
 
+    def _plot_colorbar(self, kws):
```

Comment:
can you make ax a named arg here and so you would pass as ``**kwds`` when you call.